### PR TITLE
Add issue template for validation requests

### LINKED ISSUE #291: Create an issue template for people needing help with validation
This could include:

- A polite notice saying this may take time on our side
    - Invite random readers to help with this issue if they can
- A link to the validation section in the README
- Add `help-wanted` and (to be created) `validation-issue` labels

Also add a bullet in the 10 mins. contrib section in README to say "IF you know how to validate, help someone out!" w/ link to label

### PR DESCRIPTION
**Pull request by sdruskat**
**Related issues**

Fix #291

**Describe the changes made in this pull request**

- Add an issue template for users who have issues with validation

**Instructions to review the pull request**

- Look at the new issue template file

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/30_validation_issue.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Invalid `CITATION.cff` file
+
+```yaml
+Paste your invalid CITATION.cff file here!
+```
+
+## Context
+
+- Please describe what you want to do (create a valid `CITATION.cff` file, show the citation information in the sidebar of your GitHub repository, ...)
```
Comment (jspaaks) [inline .github/ISSUE_TEMPLATE/30_validation_issue.md:25]:
```suggestion
- Please describe what you want to do (create a valid `CITATION.cff` file, show the citation information in the sidebar of your GitHub repository, ...)

```

**Review by jspaaks — APPROVED** [review]
Good stuff, thank you

File: .github/ISSUE_TEMPLATE/30_validation_issue.md (hunk 1 above)
Comment (sdruskat) [reply .github/ISSUE_TEMPLATE/30_validation_issue.md:25]: Weird, I have a blank line at the end in my local file ... Anyhow...

**Comment by sdruskat** [comment]
> Good stuff, thank you. Maybe we should copy this to main as well?

I'd say not just yet, as it links to the new validation section in the README which won't be in `main` yet. Think it's fine to wait a few more days...

**Pull request merged**